values-zh-rTW/strings.xml

```diff
@@ -2133,391 +2133,391 @@
-    <string name="settings_category_controls">Controls</string>
-    <string name="draft_cat_rci_parts00_title">Manual building</string>
-
-
-    <string name="settings_verticaltoolbuttons">Vertical tool actions</string>
-    <string name="toolaction_build">Build</string>
-    <string name="toolaction_joinroad">Join roads</string>
-    <string name="toolaction_trafficlights">Use traffic lights</string>
-    <string name="toolaction_bridge">Bridge</string>
-    <string name="toolaction_constantheight">Constant height</string>
-    <string name="toolaction_modifyunder">Below</string>
-    <string name="toolaction_levelup">Higher</string>
-    <string name="toolaction_leveldown">Lower</string>
-    <string name="toolaction_upwards">Upwards</string>
-    <string name="toolaction_downwards">Downwards</string>
-    <string name="toolaction_nextframe">Next variant</string>
-    <string name="toolaction_prevframe">Previous variant</string>
-    <string name="toolaction_singlelane">Single lane</string>
-    <string name="toolaction_randomcar">Select random</string>
-    <string name="toolaction_hidebuildings">Highlight car</string>
-    <string name="toolaction_radiusbigger">Bigger</string>
-    <string name="toolaction_radiussmaller">Smaller</string>
-    <string name="toolaction_routeremove">Remove</string>
+    <string name="dialog_reported_text">感謝你的檢舉。 我們將對其進行審查並採取相應的行動。\n如果需要，你可以將更多資訊傳送至report@theotown.com</string>
+    <string name="dialog_report_cmd">檢舉/報告</string>
+    <string name="stage_comment_reportsubject">由「%1$s」發送的評論</string>
+    <string name="stage_store_reportsubject">插件「%1$s」</string>
+    <string name="stage_comment_noperm">你不允許對此插件發送評論。</string>
+    <string name="store_modecommented">評論</string>
+    <string name="report_sexual">色情內容</string>
+    <string name="report_violent_or_repulsive">暴力或令人厭惡的內容</string>
+    <string name="report_hateful_or_abusive">仇恨或辱罵性內容</string>
+    <string name="report_dangerous">危險內容</string>
+    <string name="report_promotes_terrorism">宣揚恐怖主義</string>
+    <string name="report_spam_or_misleading">刷屏或誤導</string>
+    <string name="report_infringes_my_rights">侵犯我的權利</string>
+    <string name="report_bad_behavior">不良行為</string>
+    <string name="report_violates_store_rules">違反其他商店規則</string>
+    <string name="stage_comment_cmdsend">發送</string>
+    <string name="stage_comment_cmdsendhint">在這裡鍵入你的評論⋯</string>
+    <string name="dialog_storerestart_cmdlater">稍後</string>
+    <string name="wealthy_poor">貧窮階級</string>
+    <string name="wealthy_medium">中產階級</string>
+    <string name="wealthy_rich">富裕階級</string>
+    <string name="ci_rci_res">住宅區</string>
+    <string name="ci_rci_com">商業區</string>
+    <string name="ci_rci_ind">工業區</string>
+    <string name="settings_supportdrawingintwomodetools">支援在雙模式工具中拖動</string>
+    <string name="settings_category_controls">控制</string>
+    <string name="draft_cat_rci_parts00_title">人工建築</string> <!-- May needs correction, English is "Manual Buidling"-->
```

Comment:
@LobbyDivinus do strings support xml comments?

Reply:
From the standards document:
http://www.w3.org/TR/REC-xml/#sec-comments

> [Definition: **Comments** may appear anywhere in a document outside other [markup](https://www.w3.org/TR/REC-xml/#dt-markup); in addition, they may appear within the document type declaration at places allowed by the grammar. They are not part of the document's [character data](https://www.w3.org/TR/REC-xml/#dt-chardata); an XML processor MAY, but need not, make it possible for an application to retrieve the text of comments. [For compatibility](http://www.w3.org/TR/REC-xml/#dt-compat), the string " -- " (double-hyphen) MUST NOT occur within comments.] Parameter entity references MUST NOT be recognized within comments.